ontology/EBUCore/ebucore.owl

```diff
@@ -4145,7 +4041,8 @@ ec:noiseFilter rdf:type owl:DatatypeProperty ,
 ec:normalDuration rdf:type owl:DatatypeProperty ;
                   rdfs:domain ec:NormalPlayTime ;
                   rdfs:range xsd:string ;
-                  rdfs:comment "a duration expressed as a string formated as ISO8601 ie PT3M23S"@en .
+                  dcterms:description "a duration expressed as a string formated as ISO8601 ie PT3M23S"@en ;
+                  rdfs:label "Normal duration"@en .
```

Comment:
What does "normal" mean in this context?

Reply:
We have called the start time and the duration expressed using ISO8601 for normal, but perhaps we should come up with a better name later.